src/main/java/de/doubleslash/keeptime/view/ReportController.java

```diff
@@ -87,130 +96,208 @@
 
    private static final Logger LOG = LoggerFactory.getLogger(ReportController.class);
 
-   private DatePicker datePicker; // for calendar element
-
    private Model model;
 
    private Controller controller;
 
+   private Stage stage;
+
    private ColorTimeLine colorTimeLine;
 
+   private LocalDate currentReportDate;
+
    @FXML
    private void initialize() {
       LOG.info("Init reportController");
+      currentReportDate = LocalDate.now();
+      initTableView();
+      colorTimeLine = new ColorTimeLine(colorTimeLineCanvas);
+   }
 
-      this.datePicker = new DatePicker(LocalDate.now());
-      this.datePicker.valueProperty().addListener((observable, oldvalue, newvalue) -> {
-         LOG.info("Datepicker selected value changed to {}", newvalue);
-         updateReport(newvalue);
-      });
+   private void initTableView() {
+      final TreeTableColumn<TableRow, String> noteColumn = new TreeTableColumn<>("Notes");
```

Comment:
the notes are not wrapped anymore, but shown with ...
should wrap the notes again if possible, so that they can be read
-was commentLabel.setWrapText(true);